tests/e2e/bin/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3.1'
+
+services:
+  mysql:
+    image: mysql:5.7
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: example
+      MYSQL_DATABASE: wordpress_test
+
+  wordpress_e2e_tests:
+    image: wordpress
```

Comment:
For clarity, you might want to provide image version+server type here, e.g. `5-php7.3` ("WP 5.x, PHP 7.3 FPM") or some other combination from here: https://hub.docker.com/_/wordpress/

...or just keep it to default but unsure to me if we now test e.g. on Apache or PHP-FPM?

Reply:
This is something which would be used only for local e2e tests, and really I don't have a strong opinion on what image should we use here.  

> Apache or PHP-FPM
Can you explain this a bit more? I don't really know how this might affect running WP installation and the tests itself.

Reply:
I meant that from `wordpress`  default image name it's not clear if it ends up pulling FPM version or Apache version. Both are available via https://hub.docker.com/_/wordpress/

I would expect something like `wordpress:php7.3-apache` here, just like you've pinned down the MySQL version above.

If you want to keep PHP version lax, that's fine by me but just was surprised by the disparity between `mysql` and `wordpress` images. :-)

Reply:
added a version to wordpress image as recommended